webserver/Dockerfile.insider

```diff
@@ -0,0 +1,16 @@
+FROM stefanscherer/golang-windows:1.11-10.0.17744.1001 as gobuild
+
+COPY . /code
+WORKDIR /code
+
+USER ContainerAdministrator
```

Comment:
Without this instruction the go build fails with access denied writing webserver.exe into c:/code